packages/config/config/devices/0x0129/yrd4x0-f-zw3.json

```diff
@@ -0,0 +1,87 @@
+{
+	"manufacturer": "Yale",
+	"manufacturerId": "0x0129",
+	"label": "YRD4X0-F-ZW3",
+	"description": "Assure 2 Biometric Deadbolt",
+	"devices": [
+		// YRD410-F-ZW3 (2nd Generation Assure Keyed Push Button Biometric Deadbolt)
+		{
+			"productType": "0x8107",
+			"productId": "0x49d1",
+			"zwaveAllianceId": 4936
+		},
+		// YRD420-F-ZW3 (2nd Generation Assure Keyed Touch Screen Biometric Deadbolt)
+		{
+			"productType": "0x8107",
+			"productId": "0x49d2",
+			"zwaveAllianceId": 4929
+		},
+		// YRD430-F-ZW3 (2nd Generation Assure Keyless Push Button Biometric Deadbolt)
+		{
+			"productType": "0x8107",
+			"productId": "0x49d3",
+			"zwaveAllianceId": 4934
+		},
+		// YRD450-F-ZW3 (2nd Generation Assure Keyless Touch Screen Biometric Deadbolt)
+		{
+			"productType": "0x8107",
+			"productId": "0x49d5",
+			"zwaveAllianceId": 4913
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "templates/yale_template.json#volume_two_options"
+		},
+		{
+			"#": "2",
+			"$import": "templates/yale_template.json#auto_relock"
+		},
+		{
+			"#": "3",
+			"$import": "templates/yale_template.json#auto_relock_time_180"
+		},
+		{
+			"#": "4",
+			"$import": "templates/yale_template.json#wrong_code_limit_3_to_10"
+		},
+		{
+			"#": "7",
+			"$import": "templates/yale_template.json#wrong_code_lockout_10_to_132"
+		},
+		{
+			"#": "8",
+			"$import": "templates/yale_template.json#operating_mode_normal_privacy_passage"
+		},
+		{
+			"#": "11",
+			"$import": "templates/yale_template.json#one_touch",
+			"defaultValue": 255
+		},
+		{
+			"#": "12",
+			"$import": "templates/yale_template.json#privacy_button"
+		},
```

Comment:
This one is not documented in the manual

Reply:
Removed